.circleci/config.yml

```
@@ -959,7 +959,12 @@ commands:
 
       - run:
           name: Build main image
-          command: make docker-build-main-image ROX_PRODUCT_BRANDING="<< parameters.branding >>"
+          command: |
+            if [[ "<< parameters.branding >>" == "STACKROX_BRANDING" ]]; then
+              make docker-build-main-image ROX_PRODUCT_BRANDING="<< parameters.branding >>" ROX_IMAGE_FLAVOR="opensource"
+            else
+              make docker-build-main-image ROX_PRODUCT_BRANDING="<< parameters.branding >>"
+            fi
```

This looks good, but I want to post an alternative and see what you think:

```suggestion
          command: |
            PARAMS=(ROX_PRODUCT_BRANDING="<< parameters.branding >>")
            if [[ "<< parameters.branding >>" == "STACKROX_BRANDING" ]]; then
              PARAMS+=(ROX_IMAGE_FLAVOR="opensource")
            fi
            make docker-build-main-image "${PARAMS[@]}"
```